src/condition.cpp

```diff
@@ -508,6 +511,10 @@ void ConditionAttributes::endCondition(Creature* creature)
 			player->sendStats();
 		}
 	}
+
+	if (disableDefense) {
+		creature->setUseDefense(!disableDefense);
```

Comment:
This should be `creature->setUseDefense(!disableDefense);` or `creature->setUseDefense(true);`

Reply:
Yes, you are right. I got confused. because I changed canUseDefense to true by default and forgot about these.

First comment is right too.

Reply:
But second should be `creature->setUseDefense(disableDefense);` right?

Reply:
But inside that `if` it is guarantee that `disableDefense` is true, so you can just set it to `true` directly, without using variable.